### ISSUE TITLE
[bug] Icon Animations on Contact Page Do Not All Animate

### ISSUE THREAD

**Issue by kn0wmad** [Start9Labs/start9.com#25]
Linux / Firefox - All the icons shrink and turn salmon-coloured, with the exception of the Ghost (blog) icon

**Comment by spencerflagg** [comment]
[community.scss > line 82](https://github.com/Start9Labs/start9.com/blob/ce274f6c049dff254c887cb9157592d26b9945b6/src/assets/styles/community.scss#L82)

This is because the css assumes that the svg in the anchor contains paths, because the paths themselves get their scale transitioned, and their `fill`s are controlled. The Ghost svg, which was added later, displays the image via a <use> tag, which doesn't have a `fill`, but the scale can be controlled.

Alternatively, the svg could be replaced by a vector version. Ghost's logo doesn't really led itself to vector, but here's as close as I've been able to find: 

it's white on white, so you can't see it...
[invisible svg]
![ghost-logo-orb](https://github.com/Start9Labs/start9.com/assets/5099192/cd8fd4cd-8a5c-49a7-9c92-d9fca45cf509)
[/invisible svg]

**Comment by kn0wmad** [comment]
Doesn't look invisible to me, but I'm a dark mode maximalist.

**Comment by elvece** [comment]
Yes this is known. I reached out to the ghost team and unfortunately they do not have any svg icons. I decided to leave it without the animation for now until we could implement a generic animation for any image type.
